<launch>
<node pkg="rosserial_python" type="serial_node.py" name="serial_node">
    <param name="port" value="/dev/ttyACM0"/>
    <param name="baud" value="57600"/>
  </node>
<node pkg="auto_car" type="auto_driver.py" name = "auto_driver"/>
<node pkg="joy" type="joy_node" name = "joy_node"/>



</launch>
